.github/workflows/run_checks.yaml

```diff
@@ -142,7 +142,9 @@ jobs:
           cat out_base
           if [[ $(cat out_base | wc -l) != 0 ]]; then
             echo "Failed to apply patch cleanly to baseline hash"
+            echo "\`\`\`" >> out_base
```

Comment:
Does it make sense to use printf here?

Reply:
there's no need to use printf because echo by default will add a newline at the end. We need printf when we want things like `\n` to be recognized as a newline character instead of being recognized as a backslash and an n